src/components/App/custom-els/FileDrop/index.ts

```diff
@@ -36,11 +44,17 @@ function fixExtendedEvent(instance: Event, type: Function) {
 }
 
 export class FileDropEvent extends Event {
+  private _action: string;
   private _file: File;
   constructor(typeArg: string, eventInitDict: FileDropEventInit) {
     super(typeArg, eventInitDict);
     fixExtendedEvent(this, FileDropEvent);
     this._file = eventInitDict.file;
+    this._action = eventInitDict.action;
+  }
+
+  get action(): string {
```

Comment:
You can omit the type signature here I think. It's simply enough that typescript will infer it.